projects/flow.hoomd.lj/src/project.py

```diff
@@ -49,24 +42,15 @@ def initialize(job):
         hoomd.context.initialize("")
     with job:
         with hoomd.context.SimulationContext():
+            # create a simple cubic lattice
             n = int(ceil(pow(job.sp.N, 1.0 / 3)))
             assert n ** 3 == job.sp.N
             hoomd.init.create_lattice(unitcell=hoomd.lattice.sc(a=1.0), n=n)
             hoomd.dump.gsd("init.gsd", period=None, group=hoomd.group.all())
 
 
 @MyProject.operation
-@MyProject.post(estimated)
-def estimate(job):
-    "Ideal-gas estimate operation."
-    sp = job.statepoint()
-    # Calculate volume using ideal gas law
-    V = sp["N"] * sp["kT"] / sp["p"]
-    job.document["volume_estimate"] = V
-
-
-@MyProject.operation
-@MyProject.pre(initialized)
+@MyProject.pre.isfile("init.gsd")
```

Comment:
Would it be better to use `after`?
```suggestion
@MyProject.pre.after(initialize)
```

Reply:
Personally I like using `isfile` here because it seems like the most explicit way of showing how pre/post conditions work.